test/tcframe/io_manipulator/IOManipulatorTests.cpp

```diff
@@ -19,40 +19,85 @@ class IOManipulatorTests : public Test {
     vector<int> V;
     vector<vector<int>> M;
 
-    IOManipulator* manipulator;
+    IOManipulator* manipulatorEndsWithScalar;
```

Comment:
- `EndsWithX` -> `WithXLast` (the focus here is what the "last" variable we encounter, not what the segment ends with)
- `Grid` -> `Matrix`